plugins/cue_points/annotation/lib/api/KalturaAnnotation.php

```diff
@@ -147,4 +147,37 @@ public function validateForUpdate($sourceObject, $propertiesToSkip = array())
 			
 		return parent::validateForUpdate($sourceObject, $propertiesToSkip);
 	}
+
+    /*
+ * @param string $cuePointId
+ * @throw KalturaAPIException - when parent annotation doesn't belong to the same entry, or parent annotation
+ * doesn't belong to the same entry
+ */
+    public function validateParentId($cuePointId = null)
+    {
+
+        if (!$this->parentId)
```

Comment:
Logic is reversed, should check only if there is value

Reply:
fixed